entry_types/scrolled/package/src/entryState/legalInfo.js

```diff
@@ -0,0 +1,81 @@
+import {useEntryMetadata} from "./metadata";
+import {useEntryState} from "./EntryStateProvider";
+import {getItems} from "../collections";
+import {useMemo} from "react";
+
+/**
+ * Returns a string (comma-separated list) of copyrights of
+ * all images used in the entry.
+ * If none of the images has a rights attribute configured,
+ * it falls back to the default file rights of the entry's account,
+ * otherwise returns an empty string
+ *
+ * @example
+ *
+ * const fileRights = useFileRights();
+ * fileRights // => "author of image 1, author of image 2"
+ */
+export function useFileRights() {
+  const entryState = useEntryState();
+
+  const defaultFileRights = entryState.config.defaultFileRights;
+  const imageFiles = getItems(entryState.collections, 'imageFiles');
+  const imageFileRights = imageFiles.map(function (imageConfig) {
+    return imageConfig.rights ? imageConfig.rights.trim() : undefined;
+  }).filter(Boolean).join(', ');
```

Comment:
The original Ruby helper included file rights for other file types - even if it's called "Bildrechte" in the UI.

Reply:
Ok, I added a note to add those later.